tests/conftest.py

```diff
@@ -7,10 +7,39 @@
 from time import sleep
 
 import pytest
+from aiosmtpd.controller import Controller
 
 from pocketbase import PocketBase
 from tests.prep import ensure_pocketbase_executable
 
+SMTP_PORT = 8025
+SMTP_HOST = "localhost"
+
+
+class MessageHandler:
+    def __init__(self):
+        self.messages = []
+
+    async def handle_DATA(self, _server, _session, envelope):
+        self.messages.append(envelope)
+        return "250 Message accepted for delivery"
+
+
+@pytest.fixture(scope="function")
```

Comment:
Is scope function the best choice here? How heavy is it to spin up? Maybe better to spin the smpt server up once and just capture the messages in a function scoped fixture.

Reply:
It cost nearly nothing. I did not notice any delay in the `ci.bash` script.